I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 275, in script
    python:
ScriptError: could not find label 'start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\admin_machine\Documents\renpy-8.1.3-sdk\renpy\bootstrap.py", line 275, in bootstrap
    renpy.main.main()
  File "C:\Users\admin_machine\Documents\renpy-8.1.3-sdk\renpy\main.py", line 670, in main
    run(restart)
  File "C:\Users\admin_machine\Documents\renpy-8.1.3-sdk\renpy\main.py", line 144, in run
    renpy.execution.run_context(True)
  File "C:\Users\admin_machine\Documents\renpy-8.1.3-sdk\renpy\execution.py", line 955, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 275, in script
    python:
  File "C:\Users\admin_machine\Documents\renpy-8.1.3-sdk\renpy\script.py", line 1013, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start'.

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
Luleå 1.0
Sun Dec 31 16:01:17 2023
